### PR TITLE
WIP/Content edits to form intro pages

### PR DESCRIPTION
**Pull request by peggygannon**
Content updates to form introduction pages.

Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3889

### CODE REVIEW

**Comment by peggygannon** [comment]
@bethpotts. I added you as reviewer for the intro pages PR, but we still aren't able to see the updates in a staging link. You can view the PR here if you like until I can get a SOCKS proxy set up for this.

**Comment by ayaleloehr** [comment]
Checking on the status of this?

**Comment by peggygannon** [comment]
@ayaleloehr Hi, Alex. I have been trying to set up a SOCKS proxy, so I can view my changes in a staging environment. I'm on Windows and still unable to get this set up. (I've been working with James Kassemi on this.)  At this point, I'm wondering if a FE dev needs to show me screenshots in a staging environment for these react apps, so I can check my updates.

**Comment by phiden** [comment]
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-1990-introduction-1504897292739
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-1990e-introduction-1504897312825
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-1990n-introduction-1504897329206
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-5490-introduction-1504897409334
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-1995-introduction-1504897428024
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-5495-introduction-1504897469868
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-pension-application-527ez-introduction-1504897508651
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-burials-and-memorials-application-530-introduction-1504897517930

**Comment by peggygannon** [comment]
@bethpotts Here is the PR for the consistency edit for the form introduction pages. Sophia pulled screenshots for the pages so we have a preview look. Could you review and let me know if you have additional changes or edits to the content?

**Comment by bethpotts** [comment]
@peggygannon Here are my edits for the education forms--I know there's overlap with edits you've already made, so sorry to duplicate. I wanted to make sure I capture everything. I also realize that you know these SMEs better than I do, and all of these changes may not be possible, so I'll let you vet what I've done. I'll turn to Pension and Burial forms tomorrow.

Global Education form changes: 
- "What you'll need to fill out this application." (change "you" to "you'll")
- This bullet needs "where" removed: "Basic information about the school or training facility you want to attend." (Also, the 1990 is the only Education form that has "required" next to this bullet.
- Under No. 4, Decision, edit the first bullet to use active voice: You'll get a Certificate of Eligibility (COE), or award letter, in the mail if we approved your application.

Edits to individual forms:
**1990**
- After the warning box about not being able to save your work, add the accredited rep sentence and link: An accredited representative with a Veterans Service Organization (VSO) can help you pick the right program. [Find an accredited representative].

Note: I'm not sure which accredited rep page we should be sending people to. This is a separate issue that we need to address across the site. 
Sometime we send people here: https://www.vets.gov/disability-benefits/apply/help/index.html
And sometimes here: https://www.va.gov/ogc/apps/accreditation/index.asp

**1990E**
- Move the "Education history" bullet to be last in the list as it is for the 1990, 5490, 1995, and 5495.
- Why do we not have "required" after the "Basic information about the school" bullet as we do on the 1990? Is it not required?
- Change the accredited rep text to this: 
An accredited representative with a Veterans Service Organization (VSO) can help you pick the right program. [Find an accredited representative].
- In the warning box toward the bottom, link the whole second sentence rather than just "sign in" and move it down to it's own line (just a line break, no space between the first sentence and this second linked sentence). I'd like that second sentence to say: Sign in to be able to save your form in progress.

**1990N**
- Move the "Education history" bullet to be last in the list as it is for the forms.
- Why do we not have "required" after the "Basic information about the school" bullet as we do on the 1990? Is it not required?
- After the warning box about not being able to save your work, add the accredited rep sentence and link: An accredited representative with a Veterans Service Organization (VSO) can help you pick the right program. [Find an accredited representative].

**5490**
- Why do we not have "required" after the "Basic information about the school" bullet as we do on the 1990? Is it not required?
- Change the accredited rep text to this: 
An accredited representative with a Veterans Service Organization (VSO) can help you pick the right program. [Find an accredited representative].

**1995**
- Add "Your" to the first bullet, like this: "Your Social Security number"
- Why do we not have "required" after the "Basic information about the school" bullet as we do on the 1990? Is it not required?
- Move "Military history" down one, so it sits above "Education history."
- Is it ok/appropriate that this is the only form that, in the warning box, says, "your military and education history"? None of the others include "military" there. Should we add it to the others or remove it from here, or leave it as is.
- Change the accredited rep text to this: 
An accredited representative with a Veterans Service Organization (VSO) can help you pick the right program. [Find an accredited representative].
- In No. 3, VA review, under "What should I do while I wait?" remove the first sentence: "The transition from military to civilian life can be challenging.
- Start that same bullet with "We offer" rather than "VA offers." Remove the embedded link. Add link language at the end instead, as we do on the other forms: [Learn about career counseling options].
- The last bullet there should start with a contraction: "We'll contact you if we need more information."

**5495**
- Add "Your" to the first bullet, like this: "Your Social Security number"
- Why do we not have "required" after the "Basic information about the school" bullet as we do on the 1990? Is it not required?
- Add the accredited rep sentence and link following the first set of bullets: An accredited representative with a Veterans Service Organization (VSO) can help you pick the right program. [Find an accredited representative].
- The last bullet in No. 3 should start with a contraction: "We'll contact you if we need more information."
- In the warning box toward the bottom, link the whole second sentence rather than just "sign in" and move it down to it's own line (just a line break, no space between the first sentence and this second linked sentence). I'd like that second sentence to say: Sign in to be able to save your form in progress.

**Comment by bethpotts** [comment]
@peggygannon Here are edits for Pension and Burial form intro pages:

Pension
- Add name of form at top.
- Should we keep the PL definition of "Military history"?
- Add "Your" to first bullet under "If you're under 65...": Your work history"
- In FDC box: In the first sentence, make "there is" a contraction: "there's"
- Add a comma before the "and" in that same spot: "... to get your claim processed, and there's..."
- Under "Aid and Attendance": Change link language.
Before: Learn more about eligibility.
After: Find out if you're eligible.
- Following "Aid and Attendance": Fix the accredited rep language to what we agreed upon.
- Under "Learn about Veterans pension rates": Change link language to active voice.
Before: Find out more about how pension rates are decided.
After: Find out how we decide pension rates.
- Under VA Review: Make second line of text match other forms: either "contact you" or "let you know by mail"
- For the warning box at the end, bring the second sentence down a line and link the whole sentence.

Burial
- Add name of form at top.
- Should we keep the PL definition of "military status and history"?
- Fix the accredited rep language to what we agreed upon.
- Under "Learn about Veterans pension rates": Change link language to active voice.
Before: Find out more about how pension rates are decided.
After: Find out how we decide pension rates.
- Under VA Review: Make second line of text match other forms: either "contact you" or "let you know by mail"
- For the warning box at the end, bring the second sentence down a line and link the whole sentence.

**Comment by phiden** [comment]
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-burials-and-memorials-application-530-introduction-1505758318328
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-1990-introduction-1505758364002
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-1990e-introduction-1505758384455
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-1990n-introduction-1505758402335
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-5490-introduction-1505758436535

**Comment by phiden** [comment]
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-burials-and-memorials-application-530-introduction-1505758318328
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-1990-introduction-1505758364002
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-1990e-introduction-1505758384455
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-1990n-introduction-1505758402335
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-5490-introduction-1505758436535
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-1995-introduction-1505758471653
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-5495-introduction-1505758490495
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-pension-application-527ez-introduction-1505758531002

**Comment by peggygannon** [comment]
@bethpotts Hi, Beth, updated screenshots for the form intro pages are ^^. I did my second proof today, and noted a few things that still need fixing: 
--update to smart quotes
--Remove bullet point under "Learn more about educational programs" (for EDU forms)
--Remove bullet point under VA Review (for EDU forms)
--add updated VSO blurb to the 5495
--please note, education school info is not required for the 5490, 1990N, and the 1990E

Also, the message that you can't save a form in the 1990 and the 1990N intro pages needs to be removed. I'm working with Jeff B on this, as it involves some complex coding.

**Comment by peggygannon** [comment]
Beth's edits to the form introductions final proof listed below: 

1) I think it looks kind of strange to have bullets in 1 and 4 of the subway map, but not 2 and 3. I'm thinking that the way to solve this is not to add bullets in 2 and 3, but rather to bring up this line of text in 3 so that there's a first line in 3 that wraps. The way it is, there are two very short lines of text, each its own paragraph. Seems like a mistake, like there should be bullets for those. Instead, let's flush them together into one paragraph.
2) I avoid single bullets sitting alone. Under Learn about educational programs, can you remove that bullet? If that makes the link wrap so that "Tool" is sitting by itself on a line, which I think will happen, then bring the whole link down.
3) There's still an embedded link in the blue box at the bottom: Can you link this whole line and bump it down to its own line?: Sign in to save your form in progress.
4) Under "What if I need help filling out my application?" the link needs to be brought down a line, so the question sits alone.
5) On the Application for Pension, the last bit of text in the Prepare section says, "Learn about Veterans Pension rates." I think the P should be lowercased.

**Comment by phiden** [comment]
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-5495-introduction-1505929778681
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-burials-and-memorials-application-530-introduction-1505929519501
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-1990-introduction-1505929538296
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-1990e-introduction-1505929556128
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-1990n-introduction-1505929576490
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-1995-introduction-1505929600085
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-5490-introduction-1505929629761
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-pension-application-527ez-introduction-1505929663442

**Review by phiden — CHANGES_REQUESTED** [review]
I think we may have had some confusion -- `<p></p>` without anything in between them is not a great idea, but the closing `</p>` tag needs to be present where there's an opening one -- for example `<p><strong>What if I need help filling out my application?</strong>` really needs to have the `</p>` at the tail end.

**Comment by phiden** [comment]
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-1990-introduction-1506018987550
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-1990n-introduction-1506019100682
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-1990e-introduction-1506019131828
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-1995-introduction-1506019167253
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-5490-introduction-1506019194894
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-education-apply-for-education-benefits-application-5495-introduction-1506019211668
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-burials-and-memorials-application-530-introduction-1506019233861
screencapture-9ac58994c03284411d803554210dde64-review-vetsgov-internal-pension-application-527ez-introduction-1506019258071

**Comment by peggygannon** [comment]
@phiden PR is ready for your review. thank you for all your help with cleaning up the errant tags.

File: src/js/edu-benefits/1990/components/IntroductionPage.jsx [hunk 1]
```diff
@@ -14,48 +14,37 @@ class IntroductionPage extends React.Component {
...
       <div className="schemaform-intro">
         <FormTitle title="Apply for education benefits"/>
-        <p>This application is equivalent to Form 22-1990 (Application for VA Education Benefits).</p>
-        <SaveInProgressIntro
-          messages={this.props.route.formConfig.savedFormMessages}
-          pageList={this.props.route.pageList}
-          resumeOnly
-          {...this.props.saveInProgressActions}
-          {...this.props.saveInProgress}/>
+        <p>Equal to VA Form 22-1990 (Application for VA Education Benefits).</p>
```
Comment (phiden) [inline src/js/edu-benefits/1990/components/IntroductionPage.jsx]: Just want to be sure that the removal of the Save In Progress intro here was deliberate?

File: src/js/edu-benefits/1990n/components/IntroductionPage.jsx [hunk 2]
```diff
@@ -14,47 +14,36 @@ class IntroductionPage extends React.Component {
...
       <div className="schemaform-intro">
         <FormTitle title="Apply for education benefits under the National Call to Service program"/>
-        <p>This application is equivalent to Form 22-1990N (Application for VA Education Benefits Under the National Call to Service (NCS) Program).</p>
-        <SaveInProgressIntro
-          messages={this.props.route.formConfig.savedFormMessages}
-          pageList={this.props.route.pageList}
-          resumeOnly
-          {...this.props.saveInProgressActions}
-          {...this.props.saveInProgress}/>
+        <p>Equal to VA Form 22-1990N (Application for VA Education Benefits Under the National Call to Service (NCS) Program).</p>
```
Comment (phiden) [inline src/js/edu-benefits/1990n/components/IntroductionPage.jsx]: Want to make sure the removal of the SIP here was deliberate.

**Review by phiden — COMMENTED** [review]
Just want to make sure we're removing the SiP stuff on purpose!

File: src/js/edu-benefits/1990/components/IntroductionPage.jsx (hunk 1 above)
Comment (peggygannon) [reply src/js/edu-benefits/1990/components/IntroductionPage.jsx]: @phiden. @jbalboni I didn't remove any SIP code. Jeff B took a look at the intro files earlier this week, so he would know best if the SIP code is correct as is.

Comment (jbalboni) [reply src/js/edu-benefits/1990/components/IntroductionPage.jsx]: It's not, that should be added back. I thought it was there when I looked before, but could be wrong.

File: src/js/edu-benefits/1990n/components/IntroductionPage.jsx (hunk 2 above)
Comment (jbalboni) [reply src/js/edu-benefits/1990n/components/IntroductionPage.jsx]: Same as above, that `<SaveInProgressIntro/>` component should be added back.

**Review by phiden — APPROVED** [review]

**Pull request merged**